### ISSUE TITLE
[Events] Documentation/Cookbook

### ISSUE THREAD

**Issue by kenofori** [sschmid/Entitas-CSharp#720]
Hi,

I have an have question.

I couldnt find documentaiton/cookbook on events, their usage etc.
Adding it 'd be beneficial for any newbie like me.

Thanks

Cheers

**Comment by zhuchun** [comment]
You can find them [here](https://github.com/sschmid/Entitas-CSharp/wiki/How-I-build-games-with-Entitas-%28FNGGames%29), there's a chapter about it.

**Comment by kenofori** [comment]
Thanks @zhuchun , can you please explain for a layman whats the difference between [Event(true) and false. Where would i use which one?

**Comment by rglobig** [comment]
The `[Event(false)]` is triggered if **any** entity that has this kind of component in the group changes. `[Event(true)]` instead is only triggered if the **specific** component on the same entity, where the listener is attached, is changing. So you are basically "binding" the listener to a specific entity.

**Comment by kenofori** [comment]
Thanks :)

**Comment by zhuchun** [comment]
BTW, there's an important thing missing, check this out [Component Listener is not working](https://github.com/sschmid/Entitas-CSharp/issues/688).

**TL;DR** Add a generated `GameEventSystems` to your feature/system.
